GameData/RealSolarSystem/RemoteTech_Settings.cfg

```diff
@@ -285,6 +285,25 @@
                 }
             }
         }
+	
+	STATION
+        {
+           Guid = 5105f5a9-d628-41c6-ad4b-824c116b044e
+           Name = nz_mahia
```

Comment:
Missed that name, should be the same as the "displayName" of the KSCSwitcher patch ("NZ - Mahia") since it is a GUI string displayed in the Tracking Station/Map Views.